config/defaults.yml

```diff
@@ -4,79 +4,77 @@
 
 
 organization:
-  name: "Energy Biosciences Institute"
+  name: "Organization Name Here"
   url: #{root_path}
-  logo_file: "logo-ebi.png"
-
-site_identification_markup: "<strong>BETY</strong>db <small><strong>Biofuel Ecophysiological Traits and Yields </strong>Database</small>"
+#  logo_file: "logo-file-here.png"
 
+site_identification_markup: "<strong>GENERIC</strong>db <small><strong>Generic PEcAn, Ecological, and/or Agronomic </strong>Database</small>"
 
 # Footer
 
-footer_background_image_file: "logo-ebi-footer.jpg"
+#footer_background_image_file: "logo-footer.jpg"
 
 ## Contact Information
 admin_phone: "(000) 000-0000"
 admin_email: "admin@example.com" # a test
 
 citation_license_copyright_markup:
 
-  '<p>LeBauer, David, Rob Kooper, Patrick Mulrooney, Scott Rohde, Dan Wang, 
-  Stephen Long, Michael Dietze, (2017). BETYdb: A yield, trait, and ecosystem 
+  '
+  <p>Software: Scott Rohde, Carl Crott, David LeBauer, Patrick Mulrooney, 
+  Rob Kooper, Jeremy Kemball, Jimmy Chen, et al. pecanproject/bety: BETYdb. 
+  Zenodo, December 6, 2019. doi:10.5281/zenodo.593027</p>
+  
+  <p>LeBauer, David, Rob Kooper, Patrick Mulrooney, Scott Rohde, Dan Wang, 
+  Stephen Long, Michael Dietze, (2018). BETYdb: A yield, trait, and ecosystem 
   service database applied to second‐generation bioenergy feedstock production.
   GCB Bioenergy. doi:10.1111/gcbb.12420</p>
 
-  <p>All public data in BETYdb is made available under the <a
+  <p>All public data in this database is made available under the <a
   href="http://opendatacommons.org/licenses/by/1-0/">Open Data Commons
   Attribution License (ODC-By) v1.0.</a> You are free to share, create, and
   adapt its contents.  Data with an access_level field and value &lt;= 2 is not
   covered by this license but may be available for use with consent.</p>
 
-  <p>Copyright © 2010-2014 Energy Biosciences Institute</p>'
+'
 
 sponsors:
-    - URL: http://illinois.edu
-      title: "University of Illinois at Urbana-Champaign"
-      text: "University of Illinois at Urbana-Champaign"
+    - URL: http://example.com
```

Comment:
how about we replace this with icons and names of all universities involved that contributed to BETY
- Illinois
- BU
- AU